.infra/Freyfile.hcl

```diff
@@ -167,7 +182,6 @@ setup {
     }
     roles {
       role = "{{{init.paths.roles_dir}}}/fqdn/1.2.1"
-      fqdn = "{{lookup('env', 'FREY_DOMAIN')}}"
```

Comment:
Same concern as mentioned here: https://github.com/transloadit/uppy-server/pull/20/files#r98854186,

> I'm afraid this will break the hostname as mentioned in Slack